src/components/__tests__/DrawerHeader.cy.js

```diff
@@ -49,14 +49,39 @@ describe('<DrawerHeader>', () => {
     });
   });
 
+  it('renders help button', () => {
+    cy.dataCy('button-help').should('be.visible');
```

Comment:
Is there a reason why you are split testing `cy.dataCy('button-help')` with the `.should()` method? Also check other places in the test in this PR.

Test should be:

```js
    cy.dataCy('button-help')
      .should('be.visible')
      .should('have.css', 'font-size', '8px')
      .should('have.css', 'font-weight', '500')
      .should('have.backgroundColor', '#212121')
      .should('have.css', 'border-radius', '50%') // round
      .should('contain', 'question_mark');
```

Reply:
Edited. 
This was motivated by attempt to split certain checks by function (they also display a little more clearly in the test UI). However, it is probably neither necessary nor consistent.